packages/postgresql/src/Adaptor.js

```diff
@@ -157,23 +157,25 @@ function queryHandler(state, query, options, callback) {
  * @example
  * sql(state => `select(*) from ${state.data.tableName};`, { writeSql: true })
  * @function
- * @param {function} sqlQuery - a function which takes state and returns a
- * string of SQL.
+ * @param {string} sqlString - A sequence of characters that forms a valid SQL command.
  * @param {object} [options] - Optional options argument
  * @param {boolean} [options.writeSql] - A boolean value that specifies whether to log the generated SQL statement. Defaults to false.
  * @param {boolean} [options.execute] - A boolean value that specifies whether to execute the generated SQL statement. Defaults to false.
  * @param {function} callback - (Optional) callback function
  * @returns {Operation}
  */
-export function sql(sqlQuery, options, callback) {
+export function sql(sqlString, options, callback) {
```

Comment:
I think `sqlQuery` is still appropriate? It's a string or a function that returns a string.

I agree that the old way was too dogmatic and that making it an expandable string is correct. I just don't think we should rename it.

Also `sequence of characters that forms a valid SQL command` is strange phrasing. I'd just say "the SQL query as a string"

Reply:
Well you can pretty much write an sql command Eg: `create table tableName (column1 dataType)`,  To me `sqlQuery` means only write sql queries. Eg: `select * from patients`